packages/rn-tester/Podfile

```diff
@@ -48,9 +50,6 @@ end
 
 target 'RNTester' do
   pods()
-  if !USE_FRAMEWORKS
```

Comment:
I think that, in the process, we lost this gate. It is important because flipper doesn't work with `USE_FRAMEWORK` and, if we lose this, we could incur in other issues.